Packs/WolkenITSM/Integrations/WolkenITSM/WolkenITSM.py

```diff
@@ -0,0 +1,607 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+'''IMPORTS'''
+import traceback
+from typing import Any, Dict
+
+import urllib3
+
+# from _collections import defaultdict
+
+
+class Client(BaseClient):
+    def __init__(self, server_url, verify, proxy, headers, auth, clientId, domain, serviceAccount, refresh_token):
+        super().__init__(base_url=server_url, verify=verify, proxy=proxy, headers=headers, auth=auth)
+
+    def add_internal_notes_request(self, incidentId, clientId, domain, serviceAccount, Authorization, formData):
+
+        formData = assign_params(formData=formData)
+        headers = self._headers
+        headers.update({
+            'Authorization': 'Bearer ' + Authorization
+        })
+
+        response = self._http_request('post', f'api/v1/incidents/{incidentId}/addInternalNotes', data=formData, headers=headers)
+        return response
+
+    def create_incident_requestv1_request(self, clientId, domain, serviceAccount, Authorization, formData, file):
+        formData = assign_params(formData=formData)
+        formData = str(formData)
+        headers = self._headers
+        headers.update({
+            'Authorization': 'Bearer ' + Authorization
+        })
+
+        headers['accept'] = 'application/json'
+        print(formData)
```

Comment:
Using print is not recommended. Removing is optional, but we highly recommend to remove it.